milestone_project_1_original.py

```diff
@@ -0,0 +1,114 @@
+"""
+A simple Tic-Tac-Toe game.
+"""
+
+BOARD = [' ', ' ', ' ',
+         ' ', ' ', ' ',
+         ' ', ' ', ' ']
+
+CURRENT_PLAYER = "X"
+
+GAME_ON = True
+
+WINNER = " "
+
+
+def display_board():
+    """Display board"""
+    print("Game board      Position Guide")
+    print("  " + BOARD[0] + "|" + BOARD[1] + "|" + BOARD[2] + "             1|2|3")
+    print("  -----             -----")
+    print("  " + BOARD[3] + "|" + BOARD[4] + "|" + BOARD[5] + "             4|5|6")
+    print("  -----             -----")
+    print("  " + BOARD[6] + "|" + BOARD[7] + "|" + BOARD[8] + "             7|8|9")
+
+
+def handle_turn():
+    """Ask for player's input and place the mark on the board"""
+    print(CURRENT_PLAYER + "'s turn!")
+    position = int(input("Choose a position from 1-9: "))
+
+    position_valid = False
+
+    while not position_valid:
+        while position not in ["1", "2", "3", "4", "5", "6", "7", "8", "9"]:
+            position = input("Choose a number from 1-9: ")
+
+        position = int(position) - 1
+
+        if BOARD[position] == ' ':
+            position_valid = True
+        else:
+            print("Position taken! Please choose another number.")
+
+    BOARD[position] = CURRENT_PLAYER
+
+    display_board()
+
+
+def check_if_win():
+    """Check if there is a win"""
+    global WINNER
+    # Check rows
+    if BOARD[0] == BOARD[1] == BOARD[2] != ' ':

```

Comment:
use if-elif-else would make more sense here

Reply:
Fixed in 2d07b9c6673936279db3eb63b42d616242c0231e